pontoon/base/views.py

```diff
@@ -399,6 +399,32 @@ def search(request, template='search.html'):
     return render(request, template, data)
 
 
+def get_project_details(request):
+    """Get project locales with their pages/paths and stats."""
+    if not request.is_ajax():
+        log.error("Non-AJAX request")
```

Comment:
I don't think logging these kinds of errors is necessary, we can search request logs for this URL if we want stats on it.